studybuddies/src/environments/environment.ts

```diff
@@ -4,6 +4,10 @@
 
 export const environment = {
   production: false,
+  name: 'desarrollo',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
  name: "desarrollo",
```